.github/workflows/postrelease.yml

```diff
@@ -0,0 +1,55 @@
+name: Post Release
+on:
+  pull_request:
+    branches: [release/2.0.0-beta.*]
+    types: [closed]
```

Comment:
I don't know much about this trigger, but does "closed" differentiate between PRs being merged vs closed? I would think we would not want this to execute if we closed the PR but didn't merge it. (Although I'm not 100% sure the workflow would allow the PR to be closed and not merged.)

Reply:
Correct, there is a condition under the job itself that checks if the pr is merged. 
`if: github.event.pull_request.merged == true`
Unfortunately, there isn't a trigger for merged specifically, so even if they are closed this will run and just instantly end. I have this tested out on my fork.

Reply:
Thanks for the clarification!